packages/opentelemetry-semantic-conventions/src/trace/database.ts

```diff
@@ -19,12 +19,23 @@
  * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md
  */
 export const DatabaseAttribute = {
-  // db (required)
-  DB_TYPE: 'db.type',
+  // required
+  DB_SYSTEM: 'db.system',
   DB_INSTANCE: 'db.instance',
-  DB_STATEMENT: 'db.statement',
+  DB_STATEMENT: 'db.statement', // if applicable
+  DB_OPERATION: 'db.operation', // only required if db.statement is not applicable
   DB_URL: 'db.url',
 
-  // db (optional)
+  // optional
   DB_USER: 'db.user',
+  DB_CONNECTION_STRING: 'db.connection_string',
+
+  // Call-level attributes for specific technologies
+  DB_CASSANDRA_KEYSPACE: 'db.cassandra.keyspace',
+  DB_HBASE_NAMESPACE: 'db.hbase.namespace',
+  DB_REDIS_DATABASE_INDEX: 'db.redis.database_index', // required only if not 0
```

Comment:
Please make this comment a tsdoc style so it will show up in intellisense
```suggestion
  /** required only if not 0 */
  DB_REDIS_DATABASE_INDEX: 'db.redis.database_index',
```

Reply:
Good point, thanks @dyladan.